src/style.css

```diff
@@ -0,0 +1,89 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf) format("truetype");
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf) format("truetype");
+  font-weight: 800;
+  font-style: normal;
+}
+
+body,
+input {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Avenir", serif;
+  font-weight: 300;
+  margin: 0;
+}
+
+.inputBlock {
+  margin-top: 20px;
+}
+
+.search {
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  width: 100%;
+  border-radius: 4px;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  font-family: "Avenir", serif;
+  color: #3d4e61;
+  font-size: 16px;
+  padding: 23px 36px;
+  background-position: 26px;
+  line-height: 22px;
+  text-indent: 20px;
+}
```

Comment:
In this class, you should leave common styles for both inputs, and move big search input related to style to another class
[Checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) the 2-nd point